main.tf

```diff
@@ -316,6 +317,22 @@ module "billing_cost_management" {
   is_production = var.is_production
 }
 
+module "entitycore_svc" {
+  source = "./entitycore_svc"
+
+  aws_region                    = local.aws_region
+  vpc_id                        = local.vpc_id
+  private_alb_listener_arn      = local.private_alb_https_listener_arn
+  internet_access_route_id      = local.route_table_private_subnets_id
+  allowed_source_ip_cidr_blocks = [local.vpc_cidr_block]
+
+  entitycore_service_secrets_arn = local.entitycore_service_secrets_arn
+
+  root_path    = "/api/entitycore"
+  keycloak_url = "https://openbluebrain.com/auth/realms/SBO/"
```

Comment:
you can do: https://${local.primary_domain}/auth/realms/SBO" so it works in all enviroments.